File: lab02_stack/task01/src/publisher_node.cpp
#include "ros/ros.h"
//TODO: include the library for a msg for an integer

int main(int argc, char **argv)
{

    ros::init(argc, argv, "talker");

    //TODO: Define a nodehandle and a publisher.

    ros::Rate loop_rate(10);

    int count = 0;

    while (ros::ok())
    {

        //TODO: Create a message and publish it. Your message should be a random number between 0 and the variable 'count'

        ros::spinOnce();

        loop_rate.sleep();
        count++;
    }


    return 0;
}

File: lab02_stack/lab02_task01/scripts/publisher_node.py
#!/usr/bin/env python

import rospy
##TODO: include the library for a msg for an integer

def talker():
    ##TODO: Define a publisher for an integer
    rospy.init_node('talker', anonymous=True)
    rate = rospy.Rate(10) # 10hz
    count = 0
    while not rospy.is_shutdown():
        ##TODO: Create a message and publish it. Your message should be a random number between 0 and the variable 'count'
        rate.sleep()
        count += 1

if __name__ == '__main__':
    try:
        talker()
    except rospy.ROSInterruptException:
        pass